packages/react/src/table/docs/overview.mdx

```diff
@@ -721,3 +741,95 @@ For more information, read the [Selectable tables with batch actions pattern](/p
 	);
 };
 ```
+
+## Row with action and error
+
+To indicate an error in a row containing an action, set the `hasRowError` prop to `true`. This should be accompanied by an section alert.
```

Comment:
I think we need to call out that this is only a visual treatment for the average sighted user, and needs to be paired with a section alert. Might even need to add something to "Don't"

Reply:
Cool cool, I'll add a "Don't".

- [ ] add a don't for `invalid` state with no section alert